include/astshim/Channel.h

```diff
@@ -76,8 +76,10 @@ class Channel : public Object {
     virtual ~Channel();
 
     Channel(Channel const &) = delete;
+    /// Move constructor
     Channel(Channel &&) = default;
     Channel &operator=(Channel const &) = delete;
+    /// Move assignment operator
```

Comment:
For move assignment, it might be worth documenting whether or not the class follows the convention of leaving the source object in a valid state; I suspect the answer might be no.